internal/provider/resource_cloudflare_access_application_test.go

```diff
@@ -103,6 +105,34 @@ func TestAccCloudflareAccessApplication_WithCORS(t *testing.T) {
 	})
 }
 
+func TestAccCloudflareAccessApplication_WithSass(t *testing.T) {
+	rnd := generateRandomResourceName()
+	name := fmt.Sprintf("cloudflare_access_application.%s", rnd)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccessAccPreCheck(t)
```

Comment:
you need both of these here (sets up the mock client)

```suggestion
            testAccPreCheck(t)
			testAccessAccPreCheck(t)
```